src/WebJobs.Script.Host/packages.config

```diff
@@ -19,15 +19,15 @@
   <package id="Microsoft.Azure.ServiceBus.EventProcessorHost" version="2.2.10" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs" version="2.2.0-beta1-11229" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs.Core" version="2.2.0-beta1-11229" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions" version="2.1.0-beta4-10582" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-beta6-10582" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="2.2.0-beta1-10588" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs.Extensions.BotFramework" version="1.0.15-beta" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.1.0-beta4-10582" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="2.2.0-beta1-10588" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs.Extensions.EventGrid" version="1.0.0-beta3-10015" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.MobileApps" version="1.1.0-beta4-10582" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.NotificationHubs" version="1.1.0-beta4-10582" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="2.1.0-beta4-10582" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.Twilio" version="1.1.0-beta4-10582" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.MobileApps" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.NotificationHubs" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.Twilio" version="2.2.0-beta1-10588" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" version="2.2.0-beta1-11229" targetFramework="net471" />

```

Comment:
A few of these are incorrect. Make sure that each of the previous major versions still matches after the update. E.G. DocumentDB should be 1.2.0-beta1-10588.